### PR TITLE
add shed.yaml file

### PR DESCRIPTION
**Pull request by bernt-matthias**
Otherwise the tool is not tested (or deployed).

### CODE REVIEW

**Comment by bernt-matthias** [comment]
Ping @RJMW @Tomnl ..

**Comment by RJMW** [comment]
@bernt-matthias Thank you for conducting maintenance – we really appreciate it!

This description might need updating: https://github.com/bernt-matthias/cfm-galaxy/blob/shed.yaml/tools/cfm/.shed.yml#L3

File: tools/cfm/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Metabolomics
+description: XPath XML querying tool
```
Comment (bernt-matthias) [inline tools/cfm/.shed.yml]: Maybe like so?

```suggestion
description: Competitive Fragmentation Modeling (CFM)
```

**Comment by bernt-matthias** [comment]
You probably need to setup the testtoolshed (`TTS_API_KEY`) and toolshed (`TS_API_KEY`) API key before merging.

**Comment by RJMW** [comment]
These should already be in place, as we have previously utilized them for other galaxy tools developed by the team and collaborators.

**Comment by bernt-matthias** [comment]
> These should already be in place, as we have previously utilized them for other galaxy tools developed by the team and collaborators.

Not sure if they need to be setup per repo

**Comment by bernt-matthias** [comment]
But if you like we can just merge and see

**Comment by bernt-matthias** [comment]
Ping @RJMW

**Comment by bernt-matthias** [comment]
Thanks a lot. The deployment did work.

**Pull request merged**